internal/tasks/tasks.go

```diff
@@ -71,7 +72,7 @@ func prettyTasksSubName(ts []*task.Task) {
 	}
 }
 
-func ExecParallelTasks(taskType int, curveadm *cli.CurveAdm, dcs []*configure.DeployConfig) error {
+func ExecParallelTasks(taskType int, curveadm *cli.CurveAdm, dcs []*configure.DeployConfig, args ...string) error {
```

Comment:
We can use `curveadm.memStorage` to pass some argument  instead of add argument `args ...string`, see also [clean_service.go](https://github.com/opencurve/curveadm/blob/17d8ada19041e853407f0e7f08336363bacc8d37/internal/tasks/clean_service.go#L105).

Reply:
Good, I have Done as you said.